ruby/vampires.rb

```diff
@@ -0,0 +1,76 @@
+valid_age = true
+eats_garlic = true
+wants_ins = true
+result = "Results inconclusive."
+valid_name = true
+likes_sunshine = true
+
+puts "How many employees will be processed?"
+num_employees = gets.chomp 
+
+i = 1
+
+until i > num_employees.to_i
+
+	puts "What is your name?"
+	name = gets.chomp
+	name = name.downcase
+	if(name == "drake cula") || (name == "tu fang")
+		valid_name = false
+	end
+
+	puts "How old are you? What year were you born?"
+	age = gets.chomp
+	birth_year = gets.chomp
+
+	if (2016 - birth_year.to_i) != age.to_i

```

Comment:
with the setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.new.year`